cmd/monaco/v1/all_configs_integration_test.go

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  */
 
-package legacy
+package v1
```

Comment:
Should we maybe run all intergration tests with `-v`, it would help to see which configs are actually deployed.

Reply:
Good idea! Done in 8cd53cc